ccd-definition/CaseEventToFields/DefendantResponseAdmitPartLRspec.json

```diff
@@ -134,6 +134,27 @@
     "DisplayContext": "COMPLEX",
     "ShowSummaryChangeOption": "N"
   },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "DEFENDANT_RESPONSE_SPEC",
+    "CaseFieldID": "homeOptionsTitleLRspec",
+    "PageDisplayOrder": 13,
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "READONLY",
+    "PageID": "defendantHomeOptions",
+    "ShowSummaryChangeOption": "Y",
+    "PageShowCondition": "respondent1ClaimResponseTypeForSpec=\"PART_ADMISSION\" AND defenceAdmitPartPaymentTimeRouteRequired!=\"IMMEDIATELY\""
```

Comment:
since this is for admit part of the claim but not paid, could you please put specDefenceAdmittedRequired = \"No\" condition as well, otherwise this screen will appear on "admit part of the claim" journey.